README.md

```diff
@@ -80,9 +80,9 @@ JNA Platform
 
 [![Maven Central](https://img.shields.io/maven-central/v/net.java.dev.jna/jna-platform.svg?label=Maven%20Central)](https://search.maven.org/artifact/net.java.dev.jna/jna-platform/5.12.1/jar)&nbsp;[jna-platform-5.12.1.jar](https://repo1.maven.org/maven2/net/java/dev/jna/jna-platform/5.12.1/jna-platform-5.12.1.jar)&nbsp;[jna-platform-jpms-5.12.1.jar](https://repo1.maven.org/maven2/net/java/dev/jna/jna-platform-jpms/5.12.1/jna-platform-jpms-5.12.1.jar)
 
-This artifact holds cross-platform mappings and mappings for a number of commonly used platform
+This artifact holds cross-platform mappings and mappings for several commonly used platform
 functions, including a large number of Win32 mappings as well as a set of utility classes
-that simplify native access. The code is tested and the utility interfaces ensure that
+that simplify native access. The code is tested, and the utility interfaces ensure that
```

Comment:
Another case where I had to consult a style guide and decide whether the second phrase was an independent clause.